src/UFS2ARCO/ufsdataset.py

```diff
@@ -57,22 +58,22 @@ class UFSDataset:
 
     chunks_in = None
     chunks_out = None
-    coords = None
-    data_vars = None
-    zarr_name = None
+    coords = []
+    data_vars = []
+    zarr_name = ""
 
     @property
-    def forecast_path(self):
+    def forecast_path(self) -> str:
         """Where to write forecast data variables to"""
-        return join(self.path_out, "forecast", self.zarr_name)
+        return str(Path(self.path_out) / "forecast" / self.zarr_name)
```

Comment:
Does `os.path.join` not work for some operating system? I would have thought that joining paths as strings with "/" as a delimiter would cause a problem for windows users, and that `os.path.join` is the interoperable version. We could also use pathlib, which is supposedly a better package, but I've never used it.

Reply:
This does not join with '/'.  The '/' operator uses whatever is appropriate for the OS it is running on.  Determined at runtime.  The '/' is not a literal.

Reply:
I see now. I totally missed that there are no quotes. I didn't know it was an operator in that way, very cool